pl_bolts/callbacks/verification/batch_gradient.py

```diff
@@ -190,3 +192,18 @@ def collect_batches(tensor: torch.Tensor) -> torch.Tensor:
 
     apply_to_collection(data, dtype=torch.Tensor, function=collect_batches)
     return tensors
+
+
+@contextmanager
+def eval_batchnorm(model):
+    to_revert = []
+    try:
+        for m in model.modules():
+            if isinstance(m, (torch.nn.BatchNorm1d, torch.nn.BatchNorm2d)):
```

Comment:
How about we make this list a class constant? 
We should also include other normalization layers, like BatchNorm3d and GroupNorm, right?

then I would also rename the context manager to a general name, like `eval_normalization_layers` or `eval_norm_layers`.  What do you think?